lib/features/rules/BpmnRules.js

```diff
@@ -200,7 +206,28 @@ BpmnRules.prototype.canCopy = canCopy;
  * Utility functions for rule checking
  */
 
-function nonExistantOrLabel(element) {
+/**
+ * Checks if given element can be used for starting connection.
+ *
+ * @param  {Element} source
+ * @return {Boolean}
+ */
+function canStartConnection(element) {
+  if (nonExistingOrLabel(element)) {
+    return null;
+  }
+
+  var businessObject = element.businessObject;
+
+  return isAny(businessObject, [
```

Comment:
Missing check for element not being an end event.

Reply:
Actually, it should stay like it is. See: [#872](https://github.com/bpmn-io/bpmn-js/issues/872)

Reply:
👍